### PR TITLE
i18n(zh-tw): Update `i18n/zh-tw` and add `pages/zh-tw/install` translations

### PR DESCRIPTION
**Pull request by asgoshawk**
#### What kind of changes does this PR include?

- Translated content

#### Description

Add `src/i18n/zh-tw/docsearch.ts` and update `src/i18n/zh-tw/nav.ts` .
Add `src/pages/zh-tw/install` including `auto.md` and `manual.md` .

### CODE REVIEW

File: src/pages/zh-tw/install/manual.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+```bash
+mkdir my-astro-project
+cd my-astro-project
+```
+當你進到你的新目錄後，建立你專案的 `package.json` 檔。這將會用來管理你專案的依賴模組包括 Astro。如果你不熟悉這個檔案的格式，可執行下列的指令來建立一個。
+
+```bash
+npm init --yes
+```
```
Comment (delucis) [inline src/pages/zh-tw/install/manual.md:34]: We made this change to show the command for all three package managers:

````suggestion
<PackageManagerTabs>
  <Fragment slot="npm">
  ```shell
  npm init --yes
  ```
  </Fragment>
  <Fragment slot="pnpm">
  ```shell
  pnpm init 
  ```
  </Fragment>
  <Fragment slot="yarn">
  ```shell
  yarn init --yes
  ```
  </Fragment>
</PackageManagerTabs>
````

File: src/pages/zh-tw/install/manual.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## 2. 安裝 Astro
+
+首先，安裝 Astro 專案的依賴模組到你的專案內。
+
+```bash
+npm install astro
+```
```
Comment (delucis) [inline src/pages/zh-tw/install/manual.md:42]: And last one!

````suggestion
<PackageManagerTabs>
  <Fragment slot="npm">
  ```shell
  npm install astro
  ```
  </Fragment>
  <Fragment slot="pnpm">
  ```shell
  pnpm install astro 
  ```
  </Fragment>
  <Fragment slot="yarn">
  ```shell
  yarn add astro
  ```
  </Fragment>
</PackageManagerTabs>
````

**Review by delucis — COMMENTED** [review]
Hi @asgoshawk! Thank you so much for this translation — looks great!

Sorry we didn’t get to this sooner. In the last week we added a component for showing the different package managers (npm, yarn, pnpm) in separate tabs. I’ve made some suggestions here to update this PR with those changes but then we should be good to go!

**Comment by asgoshawk** [comment]
Hi @delucis , thank you for the reply! It's nice and clear to have the package related tab. I've update the contents with those suggestions. Please let me know if there is another issue or problem. Thanks!

**Review by delucis — APPROVED** [review]
Looks perfect — thank you for making those changes!

**Pull request merged**